modules/_labs/slider/react/lib/Slider.tsx

```diff
@@ -0,0 +1,198 @@
+import * as React from 'react';
+import {colors} from '@workday/canvas-kit-react-core';
+
+import styled from '@emotion/styled';
+
+export interface SliderProps {
+  max: number;
+  min: number;
+  startValue: number;
+  step?: number;
+  useInputRange?: boolean;
+
+  onChange?: (newValue: number) => void;
+  onStartDrag?: (newValue: number) => void;
+  onKeyDown?: (newValue: number) => void;
+  onKeyUp?: (newValue: number) => void;
+  onEndDrag?: (newValue: number) => void;
+}
+
+const Container = styled('div')({
+  position: 'relative',
+  width: '328px',
+  height: '32px',
+  margin: '0 !important',
+});
+
+const Interval = styled('div')({
+  fontSize: 'small',
+  color: colors.licorice300,
+  padding: '8px',
+});
+
+const SliderInput = styled('input')({
+  position: 'absolute',
+  outline: 'none',
+  padding: 0,
+  margin: 0,
+  border: 'none',
+  width: '328px',
+  zIndex: 1,
+  height: '100%',
+  cursor: 'pointer',
+
+  // webkit
```

Comment:
Note: need to add the designs for other browsers, currently only works for Firefox.